rust-bins/src/bin/keygen.rs

```diff
@@ -110,6 +112,8 @@ struct KeygenAr {
                 randomness."
     )]
     only_system_randomness: bool,
+    #[structopt(long = "v1", help = "Use deprecated version 1 of BLS keygen.")]
```

Comment:
I think it would be useful to elaborate that if they have generated keys with that version they **have to** use that flag during key recovery.